### ISSUE TITLE
Adding a way to manage the status value mappings

### ISSUE THREAD

**Issue by Andreag00** [knightss27/grafana-network-weathermap#42]
**Is your feature request related to a problem? Please describe.**
I have setup my dashboard using a Prometheus SNMP Exporter that gets data from my Mikrotik router. Mikrotik, as always, does things its own way and the value mappings for interface status are not 0 = DOWN and 1 and higher = UP. 
It uses these value mappings: 
1 -> UP
2 -> DOWN
3 -> Testing
4 -> Unknown
5 -> Dormant
6 -> Not present
7 -> lowerLayerDown

This causes all devices to appear as UP even when the interface is down.

**Describe the solution/feature you'd like**
What I would like to be implemented is an option to map values (per node) to the correct status and color and make the weathermap show the correct status, possibly even the possibility to have more than two different statuses. 
Basically something the value mappings option present in other panel plugins:
![image](https://user-images.githubusercontent.com/11886092/215155997-08d9a1e0-100b-44a1-b3a7-3234eda5a175.png)
